rpcs3/Emu/VFS.cpp

```diff
@@ -21,7 +21,9 @@ bool vfs::mount(const std::string& dev_name, const std::string& path)
 
 	writer_lock lock(table->mutex);
 
-	return table->mounted.emplace(dev_name, path).second;
+	table->mounted[dev_name] = path;
```

Comment:
after first boot the bdvd_dir could not be mounted again.
Ani reported something similar when trying to boot a game after canceling compilation.